File: Week6Exe/sentimental-mario-more/mario.py
from cs50 import get_int

# Get user input
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# Create loop
for i in range(height):
    for j in range(2 * height + 3):
        if i + j > height - 2 and j < height:
            print("#", end="")
        elif j < height:
            print(" ", end="")
        elif j == height + 1:
            print("  ", end="")
        elif j > height + 2 and i + j > height * 2 + 1:
            print("#", end="")
    print("")

File: Week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

int n;

// Get Height
do
{
    n = get_int("Height: ");
}
while (n < 1 || n > 8);

// Rows
for (int i = 0; i < n; i++)
    {

    // Columns
    for (int j = 0; j < n; j++)
    {
        if (i + j < n - 1)
        {
            printf(" ");
        }
        else
        {
            printf("#");
        }
    }
        printf("\n");
    }
}